clang/include/clang/CIR/LowerToLLVM.h

```diff
@@ -31,6 +33,24 @@ namespace direct {
 std::unique_ptr<llvm::Module>
 lowerDirectlyFromCIRToLLVMIR(mlir::ModuleOp mlirModule,
                              llvm::LLVMContext &llvmCtx);
+
+class CIRToLLVMGlobalOpLowering
```

Comment:
This is originally in `clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.h`. Any reason you're promoting it to the `include` top level dir?

Reply:
I think that was an accident. This is a different file with the same name but different location. I think I missed that detail when I was moving this bit of code over from the incubator project.